apps/veil/src/pages/tournament/ui/social-card-dialog.tsx

```diff
@@ -124,7 +124,6 @@ Join now 👇`;
                 Copy Image
               </Button>
               <Button actionType='accent' onClick={() => shareToX(text, url)}>
-                {/* eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- Xcom */}
```

Comment:
At some point in my tooling experimentation, one of the linters informed me this lint was unnecessary. We're excluding the veil code from lint actions in some places https://github.com/penumbra-zone/web/blob/403196c0c897a142de6d462892e653937a18e934/package.json#L23 but I'm not changing that behavior in this PR. As always, expect CI to pass.